src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,111 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import React, { Dispatch, SetStateAction } from 'react';
+import { splitPages } from '../../utils';
+
+type Props = {
+  items: string[],
+  perPage: number,
+  currentPage: number,
+  onPageChange: Dispatch<SetStateAction<number>>,
+};
+
+export const Pagination: React.FC<Props> = ({
+  items,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const book = splitPages(items, perPage);
+  const numOfPages = Object.keys(book);
+  const numLastPage = numOfPages.length;
+
+  const getNextPage = () => {
+    if (currentPage === items.length - 1) {
```

Comment:
seems you are comparing page number and length of items array(42)?
You should check if page is not last, so currentPage === lastPage?